sass/sub.sass

```diff
@@ -1,2 +1,14 @@
-body
-    background-color: #e2e2e2
\ No newline at end of file
+body {
```

Comment:
nie powinnaś używac nawiasów wąsatych { i }, bo to jest css, w sass ma znaczenie indentacja czyli spacje/nawiasy